### ISSUE TITLE
`volta pin` strips trailing newline from package.json

### ISSUE THREAD

**Issue by chancancode** [volta-cli/volta#629]
Diff after running `volta pin`

```
diff --git a/package.json b/package.json
index 44a03cd..9c3b6c3 100644
--- a/package.json
+++ b/package.json
@@ -41,5 +41,9 @@
     "prettier": "^1.19.1",
     "ts-jest": "^24.2.0",
     "typescript": "^3.6.4"
+  },
+  "volta": {
+    "node": "12.14.0",
+    "yarn": "1.21.1"
   }
-}
+}
\ No newline at end of file
```

**Comment by charlespierce** [comment]
Ho @chancancode! Which version of Volta are you using? This was fixed in #603 and released in version 0.6.7. Thanks!

**Comment by chancancode** [comment]
My bad! Thanks for the quick response!
